### DISCUSSION TITLE
v0.17.1 Beta App Launch w/ Windows x64

### DISCUSSION THREAD

**Discussion by Kryptopher40** [FreeTubeApp/FreeTube#2482] [Q&A]
Yesterday, after upgrading to 0.17.1 Beta when launching app it would be in stretched like format. If I double click the top tab bar of window it would minimize to correct screen size/resolution and work as expected.

Today, when I click the icon in tab bar it appears to launch, but doesn't. If I switch between virtual desktops (alt + tab) then I can see the app as if it's open, but can't access it. Task manager shows the app is running.

I've rebooted, uninstalled, reinstalled, disconnected external monitor, plugged it back in, etc...without resolving. I'm running Windows 11.

Any suggestions would be appreciated! TY

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi, this sounds like https://github.com/FreeTubeApp/FreeTube/issues/1938 https://github.com/FreeTubeApp/FreeTube/issues/1944#issuecomment-1030330084

**Reply by Kryptopher40** [reply]
Hello, Thanks for links, but I couldn't get it to work. I followed the instructions, but am unable to locate the necessary files with line I need to delete. I even downloaded Notepad++, but am not even sure how to use it. Is there a less technical way to approach this fix? I'm a blue collar privacy advocate and anything tech related is challenging for me! Ugh....TY

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi, no problem lets see if i can help u out. On what step of https://github.com/FreeTubeApp/FreeTube/issues/1944#issuecomment-1030330084 did u get stuck?

**Reply by Kryptopher40** [reply]
Step #2 on locating data. I reinstalled v0.17.1 just now and see the "AppData" folder I need to access to complete steps #3-#5 of #1944 (comment): "C:\Users\kryp\AppData\Local\Programs\FreeTube", but when I follow the folders thru file explorer, I don't see the "AppData" folder anywhere. I opened all the folders, but no joy. I even tried c/p the FreeTube ref link: "%APPDATA%/FreeTube", but nothing. I hope this makes sense. And thanks for the help!

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Go to your file explorer -> clear the address # bar / file path bar -> copy ℅appdata℅ and paste it. -> if this works then ur in the appdata folder... Locate Freetube folder -> if you've located the folder you can find then file in the root path.

**Reply by Kryptopher40** [reply]
I did it, but I get a pop up from File Explorer: "Windows can't find ℅appdata℅ . Check spelling and try again." I have the icon pinned to the taskbar and I verified it's downloaded/installed. I used the .exe download, perhaps I need to use the .zip? Sorry for my tech illiteracy!

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Try %appdata% i think its throwing an error because i used the wrong % sign

**Reply by Kryptopher40** [reply]
That worked! It took me a few tries to figure it out because in Notepad++, I deleted the entire line instead of specifically just "_id":"bounds". Instantly, I got a popup JavaScript error. So I uninstalled/reinstalled FT, added the entire line back in Notepad++, deleted just  "_id":"bounds" (as the tutorial literally says) saved and restarted the app. It works! Thanks again!

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Im glad it worked for u :)
